react/components/SKUSelector/components/Variation.tsx

```diff
@@ -54,13 +63,24 @@ const Variation = ({
       }--${slug(name)} flex flex-column mb7`}
     >
       <div className={`${styles.skuSelectorNameContainer} ma1`}>
-        <span
-          className={`${
-            styles.skuSelectorName
-          } c-muted-2 db t-small overflow-hidden mb3`}
-        >
-          {name}
-        </span>
+        <div className={`${styles.skuSelectorTextContainer} db mb3`}>
+          <span
+            className={`${
+              styles.skuSelectorName
+            } c-muted-1 t-small overflow-hidden`}
+          >
+            {name}
+          </span>
+          {displayImage && selectedItem && showValueNameForImageVariation && (
+            <span
+              className={`${
+                styles.skuSelectorSelectorImageValue
```

Comment:
why `Image`?

Reply:
Because its the written value of the image being showed. Like for color "black", you see a black shirt, but the variation value is "black"...

think of a better name?